### PR TITLE
new options - resourceLimits

### PR DESCRIPTION
**Pull request by rafixer**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -191,6 +192,21 @@ The timeout value (in milliseconds) for the worker to stop before sending SIGKIL
...
+
+### options.resourceLimits : <code>Object</code>
+Provides the set of JS engine resource constraints inside this Worker thread.
+
+**Kind**: instance property of [<code>WorkerNodesOptions</code>](#WorkerNodesOptions)  
+**Properties**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| maxYoungGenerationSizeMb | <code>Number</code> | The maximum size of the main heap in MB |
```
Comment (pawel-piecyk) [inline Readme.md:205]: Descriptions should be reversed: maxYoung - recently created objects, maxOld - "main heap".

File: lib/options.js [hunk 2]
```diff
@@ -115,6 +116,17 @@ class WorkerNodesOptions {
          * @default 100
          */
         this.workerStopTimeout = Number(workerStopTimeout);
+
+        /**
+         * Provides the set of JS engine resource constraints inside this Worker thread.
+         * 
+         * @type {Object}
+         * @property {Number} maxYoungGenerationSizeMb - The maximum size of the main heap in MB
```
Comment (pawel-piecyk) [inline lib/options.js:124]: Same, reversed descriptions in young & old.

**Review by pawel-piecyk — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (rafixer) [reply Readme.md]: done ;)

File: lib/options.js (hunk 2 above)
Comment (rafixer) [reply lib/options.js]: done ;)

**Review by pawel-piecyk — APPROVED** [review]

**Review by krzysiekbielicki — APPROVED** [review]

**Pull request merged**